### ISSUE TITLE
Software never finishes installing when trying to install it from the Store

### ISSUE THREAD

**Issue by saulonio** [feren-OS/Bug-Reporting-Center#223]
Trying to install packages from the Store has no effect. It gets stuck on "installing". There is no information provided as to wehther it is downloading or installing it and no way to halt it. The Store app is responsive and can be closed, but doing so and then attempting to install the same packages from the terminal returns: 

`Waiting for cache lock: Could not get lock /var/lib/dpkg/lock-frontend.`

**Comment by leodawson** [comment]
I'm having the exact same problem, anyone figure out how to fix this issue? I just installed feren-OS yesterday. I'm liking it a lot, dont want to switch back to KDE Neon, but if this cant be fixed I think I'll to have to. (im using Store, 2021.02)

**Comment by dedepapou** [comment]
@leodawson, I feel you. The interface of feren is just superb. Just for info, my problem seems to have disapeared by itself one week after the install. I'm not 100% sure as I have already installed all I needed from command line or from other stores (you can install other ubuntu base store pretty easily), so I haven't tested it a lot recently. I will redo some tests tomorrow and confirm what I just said.

**Comment by leodawson** [comment]
> @leodawson, I feel you. The interface of feren is just superb. Just for info, my problem seems to have disapeared by itself one week after the install. I'm not 100% sure as I have already installed all I needed from command line or from other stores (you can install other ubuntu base store pretty easily), so I haven't tested it a lot recently. I will redo some tests tomorrow and confirm what I just said.

Yeah, I'll try again in a week or two, for now I just installed KDE's Discover store, its works perfectly. Hopefully when Feren releases the new app store this problem will be resolved.

**Comment by xenowildfire** [comment]
I had a problem installing Steam from the store, when I searched for Steam there are 4 that shows up, Steam, Steam (Flatpak), Steam- devices and Steam installer. I tried installing Steam and it hung so I had to restart and open Synaptic Package Manager and installed in from there. I don't know why I didn't think about using apt-get install steam. After I installed that I went into the store and installed Lutris with no problems.
